php/aboutMe.php

```diff
@@ -1,64 +1,10 @@
 <?php
 
-$db = new PDO('mysql:host=127.0.0.1;dbname=Portfolio','root');
-//fetching everything from database [about me section]
-
-    /*
-    * This Function Fetches Data from the database and returns the query as an associative array.
-    *
-    * @param the $db is to input the new instance of PDO database.
-    *
-    *
-    * @return Assoc array which contains results.
```

Comment:
@return missing variable after datatype and doesn't explain what $results represents